web/elm/src/Dashboard/Dashboard.elm

```diff
@@ -675,24 +683,42 @@ update session msg =
 updateBody : Session -> Message -> ET Model
 updateBody session msg ( model, effects ) =
     case msg of
-        DragStart teamName cardId ->
-            ( { model | dragState = Models.Dragging teamName cardId }, effects )
+        DragStart card ->
+            ( { model | dragState = Models.Dragging card }, effects )
```

Comment:
Don't think it should be fixed in this PR, but I noticed a bug where the card doesn't render if I drag the boxes inside the card (i.e the jobs for pipelines or pipelines for instance groups).

https://user-images.githubusercontent.com/17577073/115045536-9796ce00-9ea4-11eb-8b4a-d5eefde737c5.mov